modules/persistent-storage-local-tolerations.adoc

```diff
@@ -52,4 +52,17 @@ To configure local volumes for scheduling on tainted nodes:
 <4> The volume mode, either `Filesystem` or `Block`, defining the type of the local volumes.
 <5> The path containing a list of local storage devices to choose from.
 
+[NOTE]
+====
+To create local persistent volumes (PV)s on master nodes, modify the YAML file and add the `LocalVolume` spec, as shown in the following example:
```

Comment:
Is this to create a PV on any control plane nodes or just tainted ones? Why is this a note instead of an optional step?

Do you mean:

```suggestion
To create local persistent volumes (PV)s on only control plane nodes, modify the YAML file and add the `LocalVolume` spec, as shown in the following example:
```

Reply:
This is to create (PV)s on tainted nodes.

Reply:
Incorporated the suggested change.

Reply:
Since I'm still subscribed to this PR, one quick suggestion!

`(PV)s` -> `(PVs)` or rewrite so that you can use the singular "persistent volume" on the first mention.

Reply:
Acked. I went with `(PVs)`.